### PR TITLE
Call the right algorithm

### LINKED ISSUE #65: Unclear how a sample makes its way from a platform collector to a ComputePressureUpdateCallback
https://wicg.github.io/compute-pressure/#platform-primitives says

> When a sample is obtained from the [platform collector](https://wicg.github.io/compute-pressure/#dfn-platform-collector), the [user agent](https://wicg.github.io/compute-pressure/#dfn-user-agent) MUST run the [8.6.4 Notify Compute Pressure Observers](https://wicg.github.io/compute-pressure/#notify-observers) algorithm a per the [8.6 Processing Model](https://wicg.github.io/compute-pressure/#processing-model)

That looks like the wrong algorithm though, as the process through which the sample collected by the platform is added to a `ComputePressureRecord`, how this `ComputePressureRecord` is created and how the relevant `ComputePressureObserver` instances are selected for a given target type are not clear.

### PR DESCRIPTION
**Pull request by kenchris**
Makes sure to call the right algorithm after getting data from the platform collector and make sure to consider the target

Closes #65

***
<a href="https://pr-preview.s3.amazonaws.com/kenchris/compute-pressure/pull/67.html" title="Last updated on Feb 25, 2022, 1:10 PM UTC (e3e50d9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/compute-pressure/67/f82b5fb...kenchris:e3e50d9.html" title="Last updated on Feb 25, 2022, 1:10 PM UTC (e3e50d9)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -601,48 +601,48 @@ <h3>Processing Model</h3>
...
-        Let |document:Document| be the [=environment settings object/responsible document=] of the [=current settings object=].
+        Let |record:ComputePressureRecord| be a new {{ComputePressureRecord}} instance.
       </li>
+      <aside class="note">
+        TODO: https://github.com/WICG/compute-pressure/issues/46
+      </aside>
       <li>
-        If |document| is not [=Document/fully active=], about these steps.
+        Let |record|.|state:PressureState| be an [=implementation-defined=] state given
+        |target| and the data available from the [=platform collector=].
```
Comment (arskama) [inline index.html:642]: wondering if "data available" should be replace by "|readings|"  would simplify the terminology.

File: index.html [hunk 2]
```diff
@@ -186,8 +186,8 @@ <h3>Sampling and Reporting Frequency</h3>
     (if possible) for privacy reasons, or ignore and fuse certain readings.
   </p>
   <p>
-    When a sample is obtained from the [=platform collector=], the [=user agent=] MUST run the [[[#notify-observers]]]
-    algorithm a per the [[[#processing-model]]]
+    When a sample is obtained from the [=platform collector=], the [=user agent=] MUST run
+    the [=queue telemetry data=] algorithm per the [[[#processing-model]]].
```
Comment (rakuco) [inline index.html:190]: The "queue telemetry data" algorithm takes a `target` that's not passed here.

File: index.html [hunk 3]
```diff
@@ -601,48 +601,48 @@ <h3>Processing Model</h3>
...
-    </p>
+  <section id="create-record">
+    <h3>Create and populate a ComputePressureRecord</h3>
     <ol class="algorithm">
       <li>
-        Let |record| be the result of running [[[#create-record]]].
-      </li>
-      <li>
-        [=Append=] it to |observer|.{{ComputePressureObserver/[[QueuedRecords]]}}.
+        Let |readings| be the latest reading of the [=platform collector=] for [=target type=] |target|.
```
Comment (rakuco) [inline index.html:632]: There's nothing in this section saying where `target` is coming from.

File: index.html [hunk 4]
```diff
@@ -601,48 +601,48 @@ <h3>Processing Model</h3>
...
       </li>
     </ol>
   </section>
-  <section>
-    <h3>Queue a ComputePressureRecord</h3>
-    <p>
-      To queue a {{ComputePressureRecord}} for a {{ComputePressureObserver}}
-      |observer:ComputePressureObserver|, given a [=document=] |document:Document|, run these steps:
-    </p>
+  <section id="create-record">
```
Comment (rakuco) [inline index.html:628]: It'd be good to have a `<dfn>` somewhere like in the section above so that we can also know what's referencing this definition.

**Review by rakuco — COMMENTED** [review]
A bit late now that this has been merged, but there are some things here that need to be fixed.

**Pull request merged**